src/ngAria/aria.js

```diff
@@ -126,6 +128,11 @@ function $AriaProvider() {
     };
   }
 
+  function isNodeOneOf(elem, nodeTypeArray) {
```

Comment:
This function used to live inside of the `ngClick` override directive below. This exposes it to the entire module.